lib/src/server/server.dart

```diff
@@ -149,7 +152,13 @@ class Server extends ConnectionServer {
     List<Service> services, [
     List<Interceptor> interceptors = const <Interceptor>[],
     CodecRegistry? codecRegistry,
-  ]) : super(services, interceptors, codecRegistry);
+    Function? responseErrorHandler,
```

Comment:
Wanted to make named arguments to avoid

```dart
Server(
  [ service ],
  [],
  null,
  (error, trace) { ... }
)
  ```
  
  But can't do without breaking change
  
  Can I deprecate that one and create a factory ?
  
  ```dart
  Server.create([service], responseErrorHandler: (error, trace) { ... });
  ```

Reply:
Yeah, named parameters sound great. Feel free to deprecate and create a `factory`.